docs/develop/java/versioning.mdx

```diff
@@ -113,31 +113,23 @@ in none of them, then they have to share the Id.
 
 :::caution
 
-Worker Versioning is currently in Pre-release, and backwards-incompatible changes are coming to the Worker Versioning APIs. For now, you need to provide dynamic configuration parameters to your Temporal Service to enable Worker Versioning:
+Worker Versioning is currently in Pre-release.
```

Comment:
I left a sample above for the typical admonition style.

I expect all the comments apply down to these other files as well.